api-admin/src/main/java/bio/terra/pearl/api/admin/service/ScheduledDataRepoExportService.java

```diff
@@ -33,44 +33,7 @@ public void pingDataRepoStatus() {
     }
   }
 
-  /* Create any missing datasets every hour or so. There is one dataset per study
-    environment. By decoupling dataset creation from ingest, we don't have to worry
-    about handling large async saga transactions. If a dataset has not yet been created, ingest
-    will simply be skipped until the dataset is ready. These operations happen frequently enough
-    that it shouldn't be a problem if the first round of ingest is delayed due to a missing
-    dataset: by the next round, it should be ready.
-  */
-  @Scheduled(timeUnit = TimeUnit.MINUTES, fixedDelay = 60, initialDelay = 1)
-  @SchedulerLock(
-      name = "DataRepoExportService.createDatasetsForStudyEnvironments",
-      lockAtMostFor = "10m",
-      lockAtLeastFor = "5m")
-  public void createStudyEnvironmentDatasets() {
-    if (isTdrConfigured()) {
-      logger.info("Creating datasets...");
-      dataRepoExportService.createDatasetsForStudyEnvironments();
-    } else {
-      logger.error(
-          "Error: Skipping TDR dataset creation, as TDR has not been configured for this environment.");
-    }
-  }
-
-  @Scheduled(timeUnit = TimeUnit.MINUTES, fixedDelay = 240, initialDelay = 1)
-  @SchedulerLock(
-      name = "DataRepoExportService.ingestStudyEnvironmentDatasets",
-      lockAtMostFor = "10m",
-      lockAtLeastFor = "5m")
-  public void ingestStudyEnvironmentDatasets() {
-    if (isTdrConfigured()) {
-      logger.info("Ingesting datasets...");
-      dataRepoExportService.ingestDatasets();
-    } else {
-      logger.error(
-          "Error: Skipping TDR dataset ingest, as TDR has not been configured for this environment.");
-    }
-  }
-
-  @Scheduled(timeUnit = TimeUnit.MINUTES, fixedDelay = 10, initialDelay = 0)
+  @Scheduled(timeUnit = TimeUnit.MINUTES, fixedDelay = 5, initialDelay = 0)
```

Comment:
I'm also dropping this to a 5 min poll time. Honestly I could probably go even lower but I want to see how 5 min feels before being too aggressive with it.